es/community/index.md

```diff
@@ -44,6 +44,12 @@ donde empezar:
   experiencias en sus desarrollos, discutir sobre el futuro de Ruby, y
   dar una bienvenida a los recién llegados a la comunidad Ruby.
 
+[Podcasts](/es/community/podcasts/)
```

Comment:
The link can be changed to `podcasts/` (relative link from current page, like in the `en` version).